tests/Doctrine/Tests/ORM/Mapping/ClassMetadataFactoryTest.php

```diff
@@ -469,22 +469,13 @@ public function testRejectsEmbeddableWithoutValidClassName(): void
     {
         $metadata = $this->createValidClassMetadata();
 
-        $metadata->mapEmbedded(
-            [
-                'fieldName'    => 'embedded',
-                'class'        => '',
-                'columnPrefix' => false,
-            ]
-        );
-
-        $cmf = $this->createTestFactory();
-
-        $cmf->setMetadataForClass($metadata->name, $metadata);
-
         $this->expectException(MappingException::class);
         $this->expectExceptionMessage('The embed mapping \'embedded\' misses the \'class\' attribute.');
-
-        $cmf->getMetadataFor($metadata->name);
+        $metadata->mapEmbedded([
+            'fieldName'    => 'embedded',
+            'class'        => '',
+            'columnPrefix' => false,
+        ]);
```

Comment:
This technically isn't a ClassMetadataFactory test anymore and belongs to the ClassMetadata tests